snuba/datasets/configuration/utils.py

```diff
@@ -95,45 +95,48 @@ def get_mandatory_condition_checkers(
     ]
 
 
+NUMBER_COLUMN_TYPES = {"UInt": UInt, "Float": Float}
+
+SIMPLE_COLUMN_TYPES = {
+    **NUMBER_COLUMN_TYPES,
+    "String": String,
+    "DateTime": DateTime,
+    "UUID": UUID,
+    "IPv4": IPv4,
+    "IPv6": IPv6,
+}
```

Comment:
what if we just made Column(s) a RegisteredClass?

Reply:
That's an interesting idea, but I think maybe should be a separate task/PR.

Reply:
for sure